src/node_metadata.h

```diff
@@ -62,7 +75,28 @@ class Metadata {
 #undef V
   };
 
+  struct Release {
+    Release();
+
+    std::string name;
+#if NODE_VERSION_IS_LTS
+    std::string lts;
+#endif  // NODE_VERSION_IS_LTS
+
+#ifdef NODE_HAS_RELEASE_URLS
+    std::string sourceUrl;
+    std::string headersUrl;
+#ifdef _WIN32
+    std::string libUrl;
+#endif  // _WIN32
+#endif  // NODE_HAS_RELEASE_URLS
+  };
+
   Versions versions;
+  Release release;
+
+  std::string arch;
+  std::string platform;
```

Comment:
Can the `extern Metadata metadata;` on L104 be `const`?

Reply:
I don't think `Metadata` itself can be `const` - the ICU data versions have to be loaded after they are initialized. Other members should be able to be made `const` though.